packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -28,6 +30,9 @@ const TradersHub = () => {
     React.useEffect(() => {
         setTimeout(() => {
             handleScroll();
+            setTimeout(() => {
+                setScrolled(true);
+            }, 500);
```

Comment:
Isn't it better to decrease the delay?  
```suggestion
            }, 100);
```

Reply:
Changed the delay to 200 over here since setting it to 100 happens so fast while 200 seemed smoother IMO